desktop/src/main/java/bisq/desktop/main/dao/bonding/BondingViewUtils.java

```diff
@@ -147,13 +162,18 @@ public void unLock(String lockupTxId, Consumer<String> resultHandler) {
 
             try {
                 if (!DevEnv.isDevMode()) {
+                    Tuple2<Coin, Integer> miningFeeAndTxSize = daoFacade.getMiningFeeAndTxSize(unlockAmount);
+                    Coin miningFee = miningFeeAndTxSize.first;
```

Comment:
Unfortunately daoFacade.getMiningFeeAndTxSize is not correctly named. It is only valid for the blind vote tx (I will rename the method in another commit). We need to use the correct tx which will be created at the unlock tx (without triggering any side effects as the user can cancel).